client/components/theme/index.jsx

```diff
@@ -121,7 +121,7 @@ const Theme = React.createClass( {
 			return this.renderPlaceholder();
 		}
 
-		const screenshotWidth = window && window.devicePixelRatio > 1 ? 680 : 340;
+		const screenshotWidth = typeof window !== 'undefined' && window.devicePixelRatio > 1 ? 680 : 340;
```

Comment:
Without the explicit `typeof` check, this gave `window is undefined` errors in server-side context.